src/module/interface/timeline/index.ts

```diff
@@ -0,0 +1,13 @@
+import { IUpdate } from "module/interface/update";
```

Comment:
@xh3b4sd thoughts on shared folder structure for types?
An alternative would be to have shared file: interface/index

currently:
- module
- - interface
- - - metric
- - - - index